src/deploy/index.js

```diff
@@ -100,6 +100,12 @@ var deploy = function(targetNames, options) {
       utils.logSuccess(clc.underline.bold("Deploy complete!"));
       logger.info();
       var deployedHosting = _.includes(targetNames, "hosting");
+      var deployedFunctions = _.includes(targetNames, "functions");
+      if (deployedFunctions) {
+        logger.info(
+          "Please note that it can take up to 30 seconds for your updated functions to propagate"
```

Comment:
```suggestion
          "Please note that it can take up to 30 seconds for your updated functions to propagate."
```

Reply:
+1 on the period, nice catch.

Reply:
fixed, good catch

Reply:
don't forget to squash and merge when all the checks go through